### PR TITLE
(chore) Use App.storage instead of AppStorage

### PR DESCRIPTION
**Pull request by fhamon**
and removed window.keys dependency only for a char.

### CODE REVIEW

File: workspace/assets/js/dev/panel.js [hunk 1]
```diff
@@ -161,7 +161,7 @@
 		var attached = false;
 
 		$(document).on('keydown', function (e) {
-			if (e.which === global.keys.escape) {
+			if (e.which === 27) {
```
Comment (nitriques) [inline workspace/assets/js/dev/panel.js]: This should use the new keys modules.

Comment (fhamon) [reply workspace/assets/js/dev/panel.js]: Yeah if we import the module only in dev. I don't think it's worth it to import a module on a prod build only to be used in an dev tool.

Comment (fhamon) [reply workspace/assets/js/dev/panel.js]: @PascalPiche wdyt ?

Comment (nitriques) [reply workspace/assets/js/dev/panel.js]: Let's wait for @PascalPiche, but it should at least be a variable so it would be === escape

Comment (PascalPiche) [reply workspace/assets/js/dev/panel.js]: convert to a variable for now. but yep. we should split the build to make a css and js extra part for dev / logged in user.

Comment (fhamon) [reply workspace/assets/js/dev/panel.js]: I have a full day soon (ishe) to work on the framwork. I'll keep up posted

Comment (nitriques) [reply workspace/assets/js/dev/panel.js]: @fhamon :+1:

**Comment by nitriques** [comment]
@fhamon Can you rebase please ?

**Comment by nitriques** [comment]
@fhamon it needs rebase now, sorry eh.

File: workspace/assets/js/dev/panel.js [hunk 2]
```diff
@@ -5,7 +5,11 @@
  */
 
 (function ($, global, undefined) {
-	
+
+	'use strict';
+
+	var ESCPAE_KEY = 27;
```
Comment (dominic-blain) [inline workspace/assets/js/dev/panel.js]: ESCAPE?

**Pull request closed without merging**